web/auth.ex

```diff
@@ -20,7 +20,11 @@ defmodule HexWeb.Auth do
     case result do
       {user, key} ->
         if Comeonin.Tools.secure_check(key.secret_second, second) do
-          {:ok, user}
+          if key.revoked_at === nil do
```

Comment:
Triple-equals is not necessary here, it is only needed when you expect integers and floats don't want `0 == 0.0`.